flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -0,0 +1,86 @@
+-- Create new ENUM type
+--
+CREATE TYPE SEX_ENUM AS ENUM ('male', 'female');
+CREATE TYPE USER_STATUS_ENUM AS ENUM ('active', 'deactivated', 'deleted');
+CREATE TYPE ORGANIZATION_STATUS_ENUM AS ENUM ('active', 'suspended', 'deleted');
+
+-- Allow generate UUID
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- Create table
+--
+CREATE TABLE IF NOT EXISTS sso_user_profile (
+	profile_id UUID NOT NULL PRIMARY KEY,
```

Comment:
Cần dùng hàm `uuid_generate_v4()` để tự sinh ID là default nếu không được truyền vào câu insert.
```suggestion
	profile_id UUID DEFAULT uuid_generate_v4() NOT NULL PRIMARY KEY,
```